constants/radialChart.tsx

```diff
@@ -0,0 +1,157 @@
+export const COUNTRIES_QTY = 174;
+
+export const INDICATORS_QTY = 36;
+
+export const LABELS_MAP = {
+  "POPULATION": {
```

Comment:
There is a file called `indicators.json` which contains all indicators with uuids.
Before migrating to this file, we should wait for Gustavo/Karl to add the category to each of those.

Reply:
Yes, this one was created before we talked about adding the category to the indicators.json file.
When they are finished with that, I'll remove the hardcoded value of the type key for the one in the indicators.json file.